apps/editorial_features/components/eoy/templates/body.jade

```diff
@@ -22,7 +23,7 @@ section.article-body__intro(data-section=0)
             .spacer
             .image
               if section.image.indexOf('mp4') > -1
-                video
+                video(autoplay=true, controls=false, muted=true, playsinline=true)
```

Comment:
👍 

should we add it [here](https://github.com/eessex/force/blob/20046741ba6c61832c4ee68d324e47b8e63b1649/apps/editorial_features/components/eoy/templates/body.jade#L42) as well?